ui/index.d.ts

```diff
@@ -1056,3 +1055,24 @@ export function createAzureSubscriptionClient<T extends IAddUserAgent>(
  * Multiple APIs with different versions can be supplied, but ideally a single backwards-compatible API is all that's necessary.
  */
 export function createApiProvider(azExts: AzureExtensionApi[]): AzureExtensionApiProvider;
+
+// tslint:disable-next-line:max-classes-per-file
+
+/**
+ * Wraps the vscode.OutputChannel to add timestamps to all messages by default.
+ * appendLine accepts optional parameters to override the date and to prepend the message with the resourceName
+ * Use append if you don't want timestamps for that output message
```

Comment:
I don't like overwriting appendLine. I'd prefer if you added another method like `appendLog` or something